features/channel_specs/channel_spec.feature

```diff
@@ -0,0 +1,216 @@
+@rails_post_6
+Feature: channel spec
+
+  Channel specs are marked by `:type => :channel` or if you have set
+  `config.infer_spec_type_from_file_location!` by placing them in `spec/channels`.
+
+  A channel spec is a thin wrapper for an ActionCable::Channel::TestCase, and includes all
+  of the behavior and assertions that it provides, in addition to RSpec's own
+  behavior and expectations.
+
+  It also includes helpers from ActionCable::Connection::TestCase to make it possible to
+  test connection behavior.
```

Comment:
Back ticks to trigger highlighting for module names would be good here, and do we want to encourage the older "magic" way of running these by highlighting it? Can we add a note about it being discouraged?

Reply:
I deal with the missing back ticks in #2131 but for the note I don't know exactly what you want. What do you mean by _"older "magic""_ ?